tox.ini

```
@@ -1,19 +1,17 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{37,38}-{linux,windows}
+envlist = py{37,38,39,310}
+isolated_build = true
 
 [gh-actions]
 python =
     3.7: py37
     3.8: py38
-
-[gh-actions:env]
-PLATFORM =
-    ubuntu-latest: linux
-    windows-latest: windows
+    3.9: py39
+    3.10: py310
 
 [testenv]
-commands = pytest -v --cov=./ --cov-report=xml
+commands = python -m pytest -v --cov --cov-report=xml
```

I find this is a nice option, so we still have colour in the CI viewing on GitHub
```suggestion
commands = python -m pytest -v --color=yes --cov --cov-report=xml
```